lib/Target/CBackend/CBackend.cpp

```diff
@@ -5670,7 +5670,7 @@ void CWriter::visitExtractValueInst(ExtractValueInst &EVI) {
   Out << ")";
 }
 
-LLVM_ATTRIBUTE_NORETURN void CWriter::errorWithMessage(const char *message) {
```

Comment:
this is becuse of this commit https://github.com/llvm/llvm-project/commit/6da3d8b19c32c76bb503b1a71fc167a0487ef200